picard/config.py

```diff
@@ -153,9 +215,10 @@ def __initialize(self):
 
         self.setAtomicSyncRequired(False)  # See comment in event()
         self.application = ConfigSection(self, "application")
-        self.setting = ConfigSection(self, "setting")
+        self.profiles = ConfigSection(self, "profiles")
+        self.setting = SettingConfigSection(self, "setting")
         self.persist = ConfigSection(self, "persist")
-        self.profile = ConfigSection(self, "profile/default")
+        # self.profile = ConfigSection(self, "profile/default")
```

Comment:
This can be removed I guess.

Reply:
Okay, but it's such a neat idea.  It's just that I couldn't see how to effectively use it with the stacked profiles solution.